03-visualization.Rmd

```diff
@@ -119,12 +119,25 @@ Now consider Figure \@ref(fig:gapminder), which plots this data for all `r nrow(
 Note that R will deal with large numbers using scientific notation.  So in the legend for "Population", 1.25e+09 is 1.25 $\times$ 10^9^ = 1,250,000,000 = 1.25 billion. 
 -->
 
-```{r gapminder, echo=FALSE, fig.cap="Life Expectancy over GDP per Capita in 2007"}
-ggplot(data = gapminder_2007, mapping = aes(x=`GDP per Capita`, y=`Life Expectancy`, size=Population, col=Continent)) +
+```{r echo=FALSE}
```

Comment:
Shouldn't we set this in `index.Rmd` in that chunk where we set all the options? It makes no material difference to the output since we have no ggplots until here, but I like the idea of setting all options in the above-metionned chunk.

Reply:
I like the idea of having `if: knitr::is_latex_output()`  `else: knitr::is_html_output()` statements. a very elegant solution to dealing with the two versions.

Reply:
We can do the same thing with plots as well if we need to use linetype instead of color to make it greyscale, for example too.